hints/buy-and-sell-crypto.md

```diff
@@ -0,0 +1,39 @@
+<br>
+<details class="hint-accordion">  
+    <summary>Recommended Time & Space Complexity</summary>
+    <p>
+    You should aim for a solution with <code>O(n)</code> time and <code>O(1)</code> space, where <code>n</code> is the size of the input array.
+    </p>
+</details>
+
+<br>
+<details class="hint-accordion">  
+    <summary>Hint 1</summary>
+    <p>
+    A brute force solution would be to iterate throgh the array with index <code>i</code>, considering it as the day to buy, and trying all possible options for selling it on the days to the right of index <code>i</code>. This would be an <code>O(n^2)</code> solution. Can you think of a better way?
+    </p>
+</details>
+
+<br>
+<details class="hint-accordion">  
+    <summary>Hint 2</summary>
+    <p>
+    From the problem, you should buy at a price and should always sell at a higher price. Can you try to fix anyone between buy and sell?
```

Comment:
I would remove "From the problem" it seems unnecessary.

> Can you try to fix anyone between buy and sell?

I think this could be more clear, i'm not sure what this means.

Reply:
Got it. I re-wrote it.